src/web/src/pages/CopyButton.css

```
@@ -0,0 +1,22 @@
+.copyCodeBtn {
+  position: absolute;
+  right: 0;
+  margin-right: 1.5rem;
+  padding: 0.4rem;
+  border: 1px solid #121d59;
+  border-radius: 0.5rem;
+  background-color: #ffffff;
+  color: #121d59;
+  transition-duration: 0.4s;
```

```suggestion
  transition-duration: 0.2s;
```
Keep it under 300ms for a snappier feel.